CHANGELOG.md

```diff
@@ -7,3 +7,14 @@
 ## [v0.1.0.0](https://github.com/freckle/amazonka-mtl/tree/v0.1.0.0)
 
 First tagged pre-release.
+
+## [v0.1.1.1](https://github.com/freckle/amazonka-mtl/tree/v0.1.1.1)
```

Comment:
This file lists versions in newest-first order, so this should go at the top, above v0.1.1.0 and before Unreleased. It should also link to a compare with the previous version. And update the compare URL in the Unreleased section to compare from the new version now.